docs/build/build-guide.md

```diff
@@ -101,27 +111,29 @@ Before diving into any one of these tracks, it is encouraged to read about
 {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} and its networks in order to gain
 context about the application you could make.
 
+[Take a look at the various development network options here.](./build-network-overview.md)
+
 ## Building Parachains
 
 {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} is canonically referred to as the
 Relay Chain. It is also considered a _layer zero_ protocol, as it enables the interoperability and
 shared security of multiple [parachains](../learn/learn-parachains.md), which are _layer one_
-protocols. Parachains connect to a relay chain using the
-[Parachains Protocol](../learn/learn-parachains-protocol.md).
+protocols. Parachains currently connect to a relay chain using the
+[Parachains Protocol](../learn/learn-parachains-protocol.md). More elaborate (or simpler) tasks
+could be constructed in the future.
 
 ![build 1](../assets/build-1.png)
 
 :::info
 
-Throughout this document, you will encounter the term **runtime** or **STF (State Transition
+Throughout this document, you may encounter the term **runtime** or **STF (State Transition
 Function)**. Both refer to the same concept, as they define how a particular system, i.e., a
 blockchain, should deal with state changes externally and internally. Both of these terms are used
-extensively in {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}, and by association,
-Substrate contexts.
+extensively in {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} and Substrate.
 
 :::
 
-Parachains open possibilities to construct complex **runtime**, or **STF** (state transition
+The Polkadot SDK opens possibilities to construct complex **runtime**, or **STF** (state transition
```

Comment:
Here, we are comparing parachains with smart contracts. Hence, including the term to retain the context. 

```suggestion
Parachains built through the Polkadot SDK, open possibilities to construct complex **runtime**, or **STF** (state transition
```